const createError = require('http-errors');

const seedEmail = async (emailData) =>{
    try {
        await 
    } catch (error) {
        throw createError
    }
}